### DISCUSSION TITLE
Need more info on creating an admin user

### DISCUSSION THREAD

**Discussion by mwunderling** [khoj-ai/khoj#664] [Q&A]
I am not that familiar with Django so maybe the issue is just me, but this "_Your admin account will automatically be created based on the admin credentials in that file, so pay attention to those._" from the docker setup section could be more detailed. From the associated Docker compose file I see only KHOJ_ADMIN_EMAIL & KHOJ_ADMIN_PASSWORD as being applicable in that context. If I were to set KHOJ_ADMIN_EMAIL=fubar@fubar.com and  KHOJ_ADMIN_PASSWORD=supersecret what would the resulting admin username and password be? Trying  "fubar', "fubar@fubar.com", "admin", or even "Admin" don't seem to get me anywhere for the username so I can't get past the login screen below.
 
![chrome_j5V6FkVqUS](https://github.com/khoj-ai/khoj/assets/22336325/4d7cd689-d1a5-4f87-a305-6cbd511d882a)

**Comment by steamboy23** [comment]
same here

**Comment by debanjum** [comment]
Hey folks, could you check [the local setup docs](https://docs.khoj.dev/get-started/setup/#local-server-start) to verify if the steps you followed match those? Also if you're trying to access Khoj over the network, please read [this note](https://docs.khoj.dev/get-started/setup/#setup-notes)

**Comment by Pathsvivi** [comment]
In short, the default username is "username@example.com", password is "password", if you haven't touched the yml file.

They are defined in the "docker-compose.yml" file under the line "environment". You can also create your custom account by editing that part or you can login with default account & password to the admin pannel and add new accounts in the "users" menu.
![screenshot](https://github.com/khoj-ai/khoj/assets/8099973/f7ec86d9-b2f2-4522-808d-f077381f5551)

@debanjum  By the way, the document is not straightforward for people have no backgrounds of installing docker compose by saying: "Your admin account will automatically be created based on the admin credentials in that file, so pay attention to those.".

**Comment by Sabdot33** [comment]
I am having the same issue, i am hosting on a different machine and accessing the server from another PC on the same network and i have `KHOJ_NO_HTTPS=True` and `KHOJ_DOMAIN=<IP>` in my yml file, aswell as 
`KHOJ_ADMIN_EMAIL=username@example.com`
`KHOJ_ADMIN_PASSWORD=password`
and i've tried username@example.com and password or admin and admin or even default (shown as the user on webinterface) and admin but somehow nothing seems to work...

**Comment by FelixHuoEZ** [comment]
Same error here, cannot login with default email and password, also "admin" and default password.
